app/views/manage/events/_form.html.haml

```diff
@@ -0,0 +1,10 @@
+= bs_horizontal_simple_form_for @event, url: url_for(action: @event.new_record? ? "create" : "update", controller: "events") do |f|
+  .col-8.mx-auto
+    .form-inputs
+      = f.input :title
+      = f.input :description
+      = f.input :location
+      = f.input :start
```

Comment:
Anyway we can get the hours to be in 12hr format? This would match full calendar's display format of 12hr format.

Reply:
Simple form does not support 12hr datetime picker and as mentioned above datetime-local doesn't work so lets leave this in and make a custom datetime picker or do #410 for 2.2